src/Element.cc

```diff
@@ -941,6 +1109,18 @@ ElementPtr Element::GetElement(const std::string &_name)
   return result;
 }
 
+/////////////////////////////////////////////////
+ElementPtr Element::GetElement(const std::string &_name, sdf::Errors &_errors)
+{
+  ElementPtr result = this->GetElementImpl(_name);
```

Comment:
This is duplicating code. See my comments on `AddValue`.

Reply:
Removed duplicating code.